src/etc/inc/openvpn.auth-user.php

```diff
@@ -186,15 +186,13 @@ function getNasIP() {
 }
 
 if (isset($attributes['framed_ip'])) {
-/* XXX: only use when TAP windows driver >= 8.2.x */
-/*	if (isset($attributes['framed_mask'])) {
+	if (isset($attributes['framed_mask'])) {
 		$content .= "topology subnet\n";
 		$content .= "ifconfig-push {$attributes['framed_ip']} {$attributes['framed_mask']}";
 	} else {
-*/
-	$content .= "topology net30\n";
-	$content .= "ifconfig-push {$attributes['framed_ip']} ". long2ip((ip2long($attributes['framed_ip']) + 1));
-//	}
+		$content .= "topology net30\n";
+		$content .= "ifconfig-push {$attributes['framed_ip']} ". long2ip((ip2long($attributes['framed_ip']) + 1));
```

Comment:
Yeah, definitely makes sense.

@ermal - any particular reason here why's it done the other way round for RADIUS?